test/hummingbot/strategy/cross_exchange_market_making/test_cross_exchange_market_making_gateway.py

```diff
@@ -0,0 +1,1017 @@
+import asyncio
+import unittest
+from decimal import Decimal
+from math import ceil
+from typing import List, Union
+from unittest.mock import patch
+
+import pandas as pd
+from aiounittest import async_test
+from nose.plugins.attrib import attr
+
+from hummingbot.connector.connector_base import ConnectorBase
+from hummingbot.connector.exchange.paper_trade.paper_trade_exchange import QuantizationParams
+from hummingbot.connector.test_support.mock_paper_exchange import MockPaperExchange
+from hummingbot.core.clock import Clock, ClockMode
+from hummingbot.core.data_type.common import OrderType, TradeType
+from hummingbot.core.data_type.limit_order import LimitOrder
+from hummingbot.core.data_type.order_book import OrderBook
+from hummingbot.core.data_type.order_book_row import OrderBookRow
+from hummingbot.core.data_type.trade_fee import AddedToCostTradeFee, TokenAmount
+from hummingbot.core.event.event_logger import EventLogger
+from hummingbot.core.event.events import (
+    BuyOrderCompletedEvent,
+    BuyOrderCreatedEvent,
+    MarketEvent,
+    OrderBookTradeEvent,
+    OrderFilledEvent,
+    SellOrderCompletedEvent,
+    SellOrderCreatedEvent,
+)
+from hummingbot.core.network_iterator import NetworkStatus
+from hummingbot.core.utils.tracking_nonce import get_tracking_nonce
+from hummingbot.strategy.cross_exchange_market_making.cross_exchange_market_making import (
+    CrossExchangeMarketMakingStrategy,
+    LogOption,
+)
+from hummingbot.strategy.maker_taker_market_pair import MakerTakerMarketPair
+from hummingbot.strategy.market_trading_pair_tuple import MarketTradingPairTuple
+
+ev_loop: asyncio.AbstractEventLoop = asyncio.get_event_loop()
```

Comment:
Same comment about the event loop here. It should not be created nor stored at the module level.

Reply:
ok, but this is how it's done in the AMM Arb tests